src/controllers/correctness.ts

```diff
@@ -1,172 +1,124 @@
 import { Octokit } from "@octokit/rest";
 import { ESLint } from "eslint";
-import { execSync,exec } from "child_process";
-/* import {config} from "dotenv"; */
+import { config } from "dotenv";
+import fs from 'fs';
+import path from 'path';
+import { execSync } from 'child_process';
```

Comment:
you said you had to tweak it still so update file once it's done. Also need to make tests for this file but just do simple ones.